### ISSUE TITLE
[feature] enable custom paths to split on

### ISSUE THREAD

**Issue by gabrielcsapo** [mbutki/ember-vendor-split#41]
# Summary

We have module paths that we would like to be split and wanted to support this as a function.

**Comment by gabrielcsapo** [comment]
We need this as a ember addon is producing content into vendor for tests that we need to have output to a different file to reduce time to load of our tests faster.

**Comment by chriskrycho** [comment]
@gabrielcsapo I think it's worth triaging “need” here carefully. I think nearly any time we put into optimizations around ember-vendor-split are effectively wasted, and should only be done when absolutely and incontrovertibly necessary; otherwise, that effort should be put into landing and unblocking the use of Embroider on apps which use this.

**Comment by runspired** [comment]
@chriskrycho tbh even with embroider the specific optimization this add-on wants to achieve is still massively beneficial. To my knowledge embroider and ember-auto-import have no corollary for signifying "rarely changed core assets". @ef4 might know otherwise though?

**Comment by gabrielcsapo** [comment]
I completely agree @runspired, given we have waited a long time to utilize embroider small changes to this addon could have benefited the ecosystem if we worked on this almost a year ago.

**Comment by gabrielcsapo** [comment]
I totally understand the reason to use embroider but with the performance cost for our application, until the ecosystem adopts the v2 format we are sort of SOL when it comes to options with how we can bundle code.
